flink-core/src/main/java/org/apache/flink/core/fs/Path.java

```diff
@@ -274,24 +274,6 @@ private String normalizePath(String path) {
 	}
 
 	/**
-	 * Checks if the provided path string contains a windows drive letter.
-	 * 
-	 * @param path
-	 *        the path to check
-	 * @param slashed
-	 *        <code>true</code> to indicate the first character of the string is a slash, <code>false</code> otherwise
-	 * @return <code>true</code> if the path string contains a windows drive letter, <code>false</code> otherwise
-	 */
-	private boolean hasWindowsDrive(String path, boolean slashed) {
```

Comment:
It does not really break it (previously private methods should not be considered), but the plugin complains anyways.
Will slightly rewrite this to make the plugin happy.